awesome_print.gemspec

```diff
@@ -1,13 +1,10 @@
-# Copyright (c) 2010-2013 Michael Dvorkin
-#
-# Awesome Print is freely distributable under the terms of MIT license.
-# See LICENSE file or http://www.opensource.org/licenses/mit-license.php
-#------------------------------------------------------------------------------
 require "rake"
 
+require 'awesome_print/version'
```

Comment:
maybe go back to `require File.dirname(__FILE__) + "/awesome_print"` for this file only?